core/src/main/scala/hydra/core/monitor/HydraMetrics.scala

```diff
@@ -73,13 +73,13 @@ object HydraMetrics {
   def addPromHttpMetric(topic: String, responseCode: String, path: String)(implicit ec: ExecutionContext): Unit = {
     incrementGauge(
       lookupKey =
-        "Ingest_topic_" + s"_${topic}_${path}",
-      metricName = "ingest_topic_response",
-      tags = Seq(
-        "topic" -> topic,
-        "responseCode" -> responseCode,
-        "path" -> path
-      )
+        s"_${topic}_${responseCode}_${path}",
```

Comment:
This is the biggest change. This will allow for metrics to be reported per path, topic, and response code. Before, when a topic would get a code, the metrics would only increment for whatever that first code was. 
For example, if a 200 response came through, the metric would increase. However if a 400 response came through, it would be counted as a 200. This had to do with the lookupKey.